ten-simple-rules-dockerfiles.Rmd

```diff
@@ -767,6 +767,10 @@ A recommended ordering based on these considerations is as follows, and you can
 5. Labels
 6. Entrypoint and default command
 
+<!-- I was going to add `docker image prune -a` here, but on -->
+<!-- reflection it does seem like that it distracts from the flow of -->
```

Comment:
Hmm, well if we take the stance of "regularly use and rebuild containers" a purge of the cache actually makes sense because you would refresh your working base (and then need to rebuild the containers you previously built). I think this would fit well in the next section within this context, and it's also good practice so your computer doesn't poop out when it fills up unexpectedly and freezes!

Reply:
@sje30 you found a supporter - do you want to add a sentence in the next section?

Reply:
Can I suggest one here:

.after "as soon as possible (cf. Rule 10)"  we could add 

"(As an aside, `docker image prune -a` is a good way to remove unused images from your system; these tend to accrue silently in your system and take up significant disk space.)"

Reply:
That would be good! I would change `-a` to `--all` so it's more explicit.

Reply:
Done.